backend.py

```diff
@@ -173,6 +172,7 @@ def die(self):
         """
         self.lives -= 1
         self.coordinates = None
+        self.unblocked_cards
```

Comment:
Nejsem si jista, co znamena tento radek - nic se nemeni, nic se nevypocitava?

Reply:
pokud zavoláš `self.unblocked_cards` tak se to přepočítá.

Reply:
Zavolani metody vyzaduje ale `()` na konci, nebo ne?

Reply:
jak se pracuje s tím když je to jako property? jen ty závorky chybí že?

Reply:
já myslím že je to takto dobře, to není metoda

Reply:
@befeleme  smazáno :)